src/NServiceBus.Persistence.AzureStorage/Config/AzureSagaPersisterConfig.cs

```diff
@@ -1,52 +0,0 @@
-namespace NServiceBus.Config
-{
-    using System.Configuration;
-
-    /// <summary>
-    /// Config section for the Azure Saga Persister
-    /// </summary>
-    public class AzureSagaPersisterConfig:ConfigurationSection
```

Comment:
Would you consider providing an `[ObsoloteEx]` obsoleting the value and giving a meaningful build error?